tflite/python/tflitePythonVivante/.vscode/settings.json

```diff
@@ -1,18 +1,21 @@
 {
-  "torizon_psswd": "",
-  "torizon_login": "",
-  "torizon_ip": "",
-  "host_ip": "",
+  "torizon_psswd": "1",
+  "torizon_login": "torizon",
```

Comment:
Actually they should be removed. You can do it by removing the "Default Device" from the IDE interface. It will automatically update this parameters: [https://developer.toradex.com/torizon/application-development/use-cases/samples/#how-to-contribute](https://developer.toradex.com/torizon/application-development/use-cases/samples/#how-to-contribute)

Reply:
I completely agree.
Sorry for that, I forgot to remove those.
Now they have been removed.